docs/book/src/quick-start.md

```diff
@@ -9,7 +9,7 @@ This Quick Start guide will cover:
 
 ## Prerequisites
 
-- [go](https://golang.org/dl/) version v1.15+ and < 1.16.
+- [go](https://golang.org/dl/) version 1.15+ and < 1.17.
```

Comment:
We need to update it for 1.16+ < 1.17. IHMO
c/c @rashmigottipati could you look in the issue which is with you about this topic?

Reply:
IMO, using go 1.15 is more suitable for kubebuilder v3.0. What about changing it to
```
- [go](https://golang.org/dl/) version v1.15+ and < 1.16(kubebuilder v3.0).   
- [go](https://golang.org/dl/) version v1.16 and < 1.17(kubebuilder v3.1).
```

Reply:
The supported version is the version that is scaffolded. See: https://github.com/kubernetes-sigs/kubebuilder/blob/v3.0.0/testdata/project-v3/go.mod#L3
I agree with your suggestion. It seems fine.

Reply:
Done.